bin/ga_script.sh

```diff
@@ -142,3 +142,12 @@ $GA_PORTAL_CKANAPI action resource_patch -c $GA_PORTAL_INI \
 	id=d22d2aca-155b-4978-b5c1-1d39837e1993 upload@"${UPLOADFILE}"
 echo 'deleted datasets done' >> "$GA_LOG_FILE"
 
+#upload archive
+cd $GA_ARCHIVE_ROOT
+cd ..
+zip -r $GA_TMP_DIR/archive.zip analytics
```

Comment:
for safety we can add a pattern to the files that we're adding to the archive. don't want to accidentally publish something that was left in this directory that we're not meaning to
```suggestion
zip -r $GA_TMP_DIR/archive.zip analytics/????-??-??/openDataPortal.siteAnalytics.*.{csv,xls}
```